src/Kdyby/Translation/DI/TranslationExtension.php

```diff
@@ -54,6 +55,15 @@ class TranslationExtension extends Nette\DI\CompilerExtension
 			self::RESOLVER_REQUEST => TRUE,
 			self::RESOLVER_HEADER => TRUE,
 		),
+		'database' => array(
+			'table' => 'translation',
+			'columns' => [
+				'key' => 'key',
+				'locale' => 'locale',
+				'translation' => 'translation'
```

Comment:
translator tomuhle říká v katalogu "messages", takže bych raději dodržel výchozí naming a pojmenoval to `message => message`